packages/gatsby-remark-embed-snippet/src/index.js

```diff
@@ -46,21 +22,32 @@ module.exports = ({ markdownAST, markdownNode }, { directory } = {}) => {
 
     if (value.startsWith(`embed:`)) {
       const file = value.substr(6)
-      const snippetPath = normalizePath(path.join(directory, file))
+      let snippetPath = normalizePath(path.join(directory, file))
+
+      // Embed specific lines numbers of a file
+      let lines = []
+      if (snippetPath.indexOf(`#L`) > -1) {
+        lines = snippetPath.match(/L\d+/g).map(l => l.replace(`L`, ``))
```

Comment:
Can we use something like https://www.npmjs.com/package/parse-numeric-range for parsing number ranges. This would also allow support for additional syntaxes, like multi ranges `1-2,5-8` for free.

So example fully syntax could be `embed:somefile.txt#L1-2,5-8`?

Reply:
I think we should match the syntax on GitHub, where each number is prefixed with `L`. The unit test for range shows that the syntax is `#L2-L4`. Using this lib we would have to strip `L` from the number range before passing it into the parser.

Reply:
I'm not convinced that sticking to GitHub syntax is desirable. 

We already use package I linked in other areas like line highlithing and hiding, so using it (if anything) will be consistent within the package.

We could conditionally support github syntax by ignoring second `L` in case of `L2-L3` syntax, but in general using those ranges syntax will support more cases than single range with totally custom range parser.